Jenkinsfile.deploy

```diff
@@ -77,59 +83,58 @@ pipeline {
                     if (env.BRANCH_NAME != "") {
                         // Switch to branch if not on a multibranch pipeline
                         env.BRANCH_NAME = params.BUILD_BRANCH_NAME
-                        dir("${WORKING_DIR}"){
-                            checkout([$class: 'GitSCM', branches: [[name: env.BRANCH_NAME]], browser: [$class: 'GithubWeb', repoUrl: "${GIT_URL}"], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: '']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'user-kie-ci10', url: "${GIT_URL}"]]])
-                        }
+                        // TODO change author
+                        checkout(githubScm.resolveRepository("kogito-cloud-operator", "radtriste", env.BRANCH_NAME, false))
                     }
```

Comment:
Isn't the deploy file will always run on master branch? and author will be kiegroup right?

Reply:
It is supposed to run on kiegroup, but I can add a `BUILD_AUTHOR` parameter
But if we have release branches that are still enabled, it needs to be configurable.

Reply:
Okay

Reply:
added the GIT_AUTHOR